QuizWhois.Domain/Entity/User.cs

```diff
@@ -5,5 +5,7 @@ public class User
         public long Id { get; set; }
 
         public string Login { get; set; }
+
+        public string Password { get; set; }
```

Comment:
so if the db was hacked all the passwords would be compromised?

Reply:
why do we need to store it at all?

Reply:
It is a leftover from previous solution, as for now we are not using login/password in any way, should be removed.